packages/headless/src/api/service/insight/query/query-response.ts

```diff
@@ -16,14 +16,14 @@ interface SearchResult {
   percentScore: number;
 }
 
-interface Facet {
+export interface Facet {
```

Comment:
This was clearly here before this PR but i'm wondering why we're defining our own facet response interface rather than using the AnyFacetResponse that's already defined and covers several facet types we'll support in the insight panel as well. @lbergeron could you help me understand? We're going to have to end up redefining very similar interfaces so just want to make sure it's worth it.

Reply:
You are bringing a very good point! I think you are right, and I should have used `AnyFacetResponse` right from the start. The good news is that we can still update `InsightQueryResponse` to use `AnyFacetResponse`.